components/Player.js

```diff
@@ -0,0 +1,60 @@
+import propTypes from 'prop-types'
+import Card from './Card'
+
+/**
+ * Player list component
+ */
+export default function Player(props) {
+  return (
+    <div className="flex md:flex-col rounded border border-slate-300 my-2">
+      <div className=" flex w-full justify-between">
+        <div className="flex ">
+          <div
+            className={`flex items-center justify-center rounded-lg p-2  ${props.bgColor}`}
+          >
+            <span className="   bg-slate-300 rounded-full h-14 w-14 ">
+              {props.letter}
```

Comment:
The letter is not showing up properly in the circle.
![image](https://user-images.githubusercontent.com/77116011/157708834-92169c79-4bc1-4962-a63c-d54f921758cf.png)